forge-gui/res/cardsfolder/s/stand_or_fall.txt

```diff
@@ -6,6 +6,6 @@ SVar:RepeatPlayer:DB$ RepeatEach | RepeatPlayers$ Opponent | RepeatSubAbility$ D
 SVar:DBTwoPiles:DB$ TwoPiles | Defined$ Player.IsRemembered | Chooser$ Player.IsRemembered | Zone$ Battlefield | ValidCards$ Creature.RememberedPlayerCtrl | Separator$ You | ChosenPile$ DBRemember | AILogic$ Best
 SVar:DBRemember:DB$ Pump | ImprintCards$ Remembered
 SVar:DBOnlyBlock:DB$ PumpAll | KW$ HIDDEN CARDNAME can't block. | ValidCards$ Creature.OppCtrl+IsNotImprinted | SubAbility$ DBCleanup
```

Comment:
Unless I'm missing something this can just be done with `IsNotRemembered` and avoid any Imprinting?

Reply:
Sounds good to me... I'll take a closer look at that one